src/patternfly/components/Form/examples/text-inputs.hbs

```diff
@@ -0,0 +1,53 @@
+{{#>group}}
+  {{#>label labelFor="textInput1"}}Standard input{{/label}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput1" inputType="text" inputName="textInput1"}}{{/text-input}}
+  {{/form-control}}
+{{/group}}
+{{#>group}}
+  {{#>label labelFor="textInput2"}}Input with placeholder{{/label}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput2" inputType="text" inputPlaceholder="Example2"}}{{/text-input}}
+  {{/form-control}}
+{{/group}}
+{{#>group}}
+  {{#>label labelFor="textInput3" labelStyleModifier="pf-is-required"}}Required input{{/label}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput3" inputType="text" required=true}}{{/text-input}}
+  {{/form-control}}
+{{/group}}
+{{#>group}}
+  {{#>label labelFor="textInput3"}}Disabled input{{/label}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput3" inputType="text" disabled="true"}}{{/text-input}}
+  {{/form-control}}
+{{/group}}
+{{#>group}}
+  {{#>label-group}}
+    {{#>label labelFor="textInput4" labelStyleModifier=" pf-is-required"}}Label with tooltip{{/label}}
+    <div class="pf-c-tooltip">
+      <i class="fas fa-info-circle"></i>
+    </div>
+  {{/label-group}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput4" inputType="text" required=true}}{{/text-input}}
+  {{/form-control}}
+  {{#>helper-text}}This is helper text{{/helper-text}}
```

Comment:
This text can be associated with the input field using `aria-describedby`. Refer to Example 2 on this page for an example: https://www.w3.org/TR/WCAG20-TECHS/ARIA1.html